SPECS/edk2/edk2.spec

```diff
@@ -0,0 +1,1086 @@
+Source12: 52-edk2-aarch64-verbose-qcow2.json
+Source13: 53-edk2-aarch64-verbose-raw.json
+
+Source20: 50-edk2-arm-verbose.json
+
+Source30: 30-edk2-ovmf-ia32-sb-enrolled.json
+Source31: 40-edk2-ovmf-ia32-sb.json
+Source32: 50-edk2-ovmf-ia32-nosb.json
+
+Source40: 30-edk2-ovmf-4m-qcow2-x64-sb-enrolled.json
+Source41: 31-edk2-ovmf-2m-raw-x64-sb-enrolled.json
+Source42: 40-edk2-ovmf-4m-qcow2-x64-sb.json
+Source43: 41-edk2-ovmf-2m-raw-x64-sb.json
+Source44: 50-edk2-ovmf-x64-microvm.json
+Source45: 50-edk2-ovmf-4m-qcow2-x64-nosb.json
+Source46: 51-edk2-ovmf-2m-raw-x64-nosb.json
+Source47: 60-edk2-ovmf-x64-amdsev.json
+Source48: 60-edk2-ovmf-x64-inteltdx.json
+
+# https://gitlab.com/kraxel/edk2-build-config
+Source80: edk2-build.py
+Source81: edk2-build.fedora
+Source82: edk2-build.fedora.platforms
+
+Source90: DBXUpdate-%{DBXDATE}.x64.bin
+Source91: DBXUpdate-%{DBXDATE}.ia32.bin
+
+Patch0001: 0001-BaseTools-do-not-build-BrotliCompress-RH-only.patch
+Patch0002: 0002-MdeModulePkg-remove-package-private-Brotli-include-p.patch
+Patch0003: 0003-MdeModulePkg-TerminalDxe-set-xterm-resolution-on-mod.patch
+Patch0004: 0004-OvmfPkg-take-PcdResizeXterm-from-the-QEMU-command-li.patch
+Patch0005: 0005-ArmVirtPkg-take-PcdResizeXterm-from-the-QEMU-command.patch
+Patch0006: 0006-OvmfPkg-enable-DEBUG_VERBOSE-RHEL-only.patch
+Patch0007: 0007-OvmfPkg-silence-DEBUG_VERBOSE-0x00400000-in-QemuVide.patch
+Patch0008: 0008-ArmVirtPkg-silence-DEBUG_VERBOSE-0x00400000-in-QemuR.patch
+Patch0009: 0009-OvmfPkg-QemuRamfbDxe-Do-not-report-DXE-failure-on-Aa.patch
+Patch0010: 0010-OvmfPkg-silence-EFI_D_VERBOSE-0x00400000-in-NvmExpre.patch
+Patch0011: 0011-CryptoPkg-OpensslLib-list-RHEL8-specific-OpenSSL-fil.patch
+Patch0012: 0012-OvmfPkg-QemuKernelLoaderFsDxe-suppress-error-on-no-k.patch
+Patch0013: 0013-SecurityPkg-Tcg2Dxe-suppress-error-on-no-swtpm-in-si.patch
+Patch0014: 0014-SecurityPkg-add-TIS-sanity-check-tpm2.patch
+Patch0015: 0015-SecurityPkg-add-TIS-sanity-check-tpm12.patch
+Patch0016: 0016-OvmfPkg-Clarify-invariants-for-NestedInterruptTplLib.patch
+Patch0017: 0017-OvmfPkg-Relax-assertion-that-interrupts-do-not-occur.patch
+
+
+# python3-devel and libuuid-devel are required for building tools.
+# python3-devel is also needed for varstore template generation and
+# verification with "ovmf-vars-generator".
+BuildRequires:  python3-devel
+BuildRequires:  libuuid-devel
+BuildRequires:  iasl
+BuildRequires:  binutils gcc git gcc-c++ make
+BuildRequires:  qemu-img
+
+%if %{build_ovmf}
+# Only OVMF includes 80x86 assembly files (*.nasm*).
+BuildRequires:  nasm
+
+# Only OVMF includes the Secure Boot feature, for which we need to separate out
+# the UEFI shell.
+# VIPERRI XXX May not be needed BuildRequires:  dosfstools
+# VIPERRI XXX May not be needed BuildRequires:  mtools
+BuildRequires:  xorriso
+
+# For generating the variable store template with the default certificates
+# enrolled.
+BuildRequires:  python3-virt-firmware >= 23.5
+
+# endif build_ovmf
+%endif
+
+%if %{cross}
```

Comment:
Are we supporting cross-compilers in Mariner?
If not can we remove this section completely?

Reply:
No. Sure I can just remove it.